Podfile

```diff
@@ -51,6 +55,12 @@ target 'WordPressAuthenticatorTests' do
   pod 'Specta', '1.0.7'
 end
 
+target 'AuthenticatorDemo' do
+  project 'Demo/AuthenticatorDemo.xcodeproj'
+
+  pod 'WordPressAuthenticator', path: '.'
+end
+
```

Comment:
I'm not too sure about the quality of this CocoaPods-based setup. But, I'm also not keen in spending too much time on it in light of our current parallel effort to adopt to SPM.

From my point of view, this setup works and lets me work on the demo app without hassle. I think it'll serve us well till the time we'll migrate to SPM.